### PR TITLE
pages: module separation

### PR DESCRIPTION
**Pull request by jmartinm**
* INCOMPATIBLE Separates pages module to independent
  Invenio-Pages package.

Signed-off-by: Javier Martin Montull <javier.martin.montull@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
Together with https://github.com/jmartinm/invenio-pages/tree/pages

Separates pages module into an independent package.

**Comment by jirikuncar** [comment]
@jmartinm I have pushed your branch jmartin/invenio-pages@pages as inveniosoftware/invenio-pages@master.

File: invenio/base/config.py [hunk 1]
```diff
@@ -74,7 +75,6 @@
     'invenio.modules.comments',
     'invenio.modules.communities',  # remove with invenio/modules/communities
     'invenio.modules.multimedia',
-    'invenio.modules.pages',
```
Comment (jirikuncar) [inline invenio/base/config.py]: IMHO we can still keep it here.

File: invenio/base/config.py [hunk 2]
```diff
@@ -63,6 +63,7 @@
 PACKAGES = [
     'invenio_records',
     'invenio_comments',
+    'invenio_pages',
```
Comment (jirikuncar) [inline invenio/base/config.py]: We should not force people to use this package as it was disabled by default. We can maybe add it to `extras_require = {..., 'pages': ['invenio-pages>=0.1.0'], ...}`.

File: invenio/base/config.py (hunk 1 above)
Comment (jmartinm) [inline invenio/base/config.py]: I can keep it, what is the use of it when the module is completely removed from Invenio?

**Pull request merged**